packages/eds-core-react/src/components/Chip/Chip.tsx

```diff
@@ -45,16 +45,18 @@ const StyledChips = styled.div.attrs<StyleProps>(
     fill: ${typography.color};
   }
 
-  @media (hover: hover) and (pointer: fine) {
-    &:hover {
-      color: ${states.hover.typography.color};
+  ${({ clickable }) => clickable && css`
```

Comment:
I see there is already a clickable check a little further down on line 70-something that only adds corsor, would it be possible to add the color and fill there instead so its in one place? also the error state has its own hover color even further down that is not checked for clickable

Reply:
ofc, and I am a little embarrassed that I did not see it myself